daffodil-core/src/main/scala/org/apache/daffodil/dsom/RawCommonRuntimeValuedPropertiesMixin.scala

```diff
@@ -21,25 +21,26 @@ import org.apache.daffodil.schema.annotation.props.PropertyMixin
 
 trait RawCommonRuntimeValuedPropertiesMixin
   extends PropertyMixin {
-  protected final lazy val optionByteOrderRaw = findPropertyOption("byteOrder")
+  final protected def forExpression = true
```

Comment:
It looks like we only ever modify that mixins that we do by hand. There are also a bunch of mixins (most of the mixes) that are autogenerated by running "sbt genManaged". Do we need to update the daffodil-propgen subproject to add these forExpression things as well where appropriate?

Reply:
Likely not, because all the runtime-valued properties are not auto-generated by propgen, but are part of a by-hand mixin. 
The generated properties are either never expressions or always expressions. The only ones we have to deal with are the ones that are sometimes expressions.

Reply:
My understanding of this exchange is that there's no change to be made at the moment. Is that correct?